definitions/npm/password-validator_v4.x.x/flow_v0.134.x-/password-validator_v4.x.x.js

```diff
@@ -0,0 +1,30 @@
+// flow-typed signature: 3587dd2cb279eaf68a55c988c6bcb495
+// flow-typed version: <<STUB>>/password-validator_v4.x.x/flow_v0.134.0
```

Comment:
Please remove those comments, they are added by the CLI.

Reply:
Updated.